NOTICE.txt

```diff
@@ -2313,6 +2289,19 @@ SOFTWARE.
 
 ---
 
+## react-native-redash
+
+This product contains 'react-native-redash' by William Candillon.
+
+The React Native Reanimated and Gesture Handler Toolbelt.
+
+* HOMEPAGE:
+  * https://github.com/wcandillon/react-native-redash
+
+* LICENSE: MIT?
```

Comment:
I assume this is MIT? I couldn't find the license details on the repo

Reply:
yes, it should be MIT, I've created an issue in the repo to see if the creator can add a license file